grails-app/assets/javascripts/asinha/payer-create.js

```diff
@@ -0,0 +1,117 @@
+function PayerCreate() {
+    
+    this.reference = $("#payer-create-container");
+    var cpfCnpjLabelReference = this.reference.find("#cpfCnpjLabel").get(0);
+    var addressInputReference = this.reference.find("#address").get(0);
+    var addressNumberInputReference = this.reference.find("#addressNumber").get(0);
+    var emailInputReference = this.reference.find("#email").get(0);
+    var provinceInputReference = this.reference.find("#province").get(0);
+    var cityInputReference = this.reference.find("#city").get(0);
+    var stateInputReference = this.reference.find("#state").get(0);
+    var cpfCnpjInputReference = this.reference.find("#cpfCnpj").get(0);
+    var phoneInputReference = this.reference.find("#phone").get(0);
+    var cepInputReference = this.reference.find("#cep").get(0);
+    var errorMessageReference = this.reference.find("#error").get(0);
+    var searchCep;
+    var _this = this;
+
+    var initInputMasks = function() {
+        new Inputmask({mask: "999.999.999-99", showMaskOnHover: false, keepStatic: true}).mask(cpfCnpjInputReference);
+        new Inputmask({mask: ["(99) 9999-9999", "(99) 9 9999-9999"], showMaskOnHover: false, keepStatic: true}).mask(phoneInputReference);
+        new Inputmask({mask: "99999-999", showMaskOnHover: false, keepStatic: true}).mask(cepInputReference);
+    };
+
+    var bindRadioButton = function() {
+        _this.reference.find("#cpfRadio").on("click", function() {
+            cpfCnpjLabelReference.innerHTML = "CPF";
+            Inputmask({mask: "999.999.999-99", showMaskOnHover: false, keepStatic: true}).mask(cpfCnpjInputReference);
+        });
+
+        _this.reference.find("#cnpjRadio").on("click", function() {
+            cpfCnpjLabelReference.innerHTML = "CNPJ";
+            new Inputmask({mask: "99.999.999/9999-99", showMaskOnHover: false, keepStatic: true}).mask(cpfCnpjInputReference);
+        });
+    };
+
+    var fillForm = (endereco) => {
+        addressInputReference.value = endereco.logradouro;
+        provinceInputReference.value = endereco.bairro;
+        cityInputReference.value = endereco.localidade;
+        stateInputReference.value = endereco.uf;
+        addressNumberInputReference.focus();
+    };
+
+    var cleanForm = () => {
+        addressInputReference.value = "";
+        provinceInputReference.value = "";
+        cityInputReference.value = "";
+        stateInputReference.value = "";
+    };
+    
+    var validateCep = function(data) {
+        let error = ("erro" in data);
+        if (!error) {
+            errorMessageReference.innerHTML = "";
+            fillForm(data);
+        } else {
+            cleanForm();
+            errorMessageReference.innerHTML = "CEP inválido";
+        };
```

Comment:
Sempre que possível utilizem earlyReturn, isso facilita a leitura e manutenção do código, por ter menos linhas de código e diminui os niveis de identação.
```suggestion
        if (error) {
            Tratativas do erro
            retur;
        }

        Tratativas do sucesso
```